app/APKServer.h

```diff
@@ -0,0 +1,18 @@
+//
+//  APKServer.h
+//  iSH
+//
+//  Created by Theodore Dubois on 11/21/20.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface APKServer : NSObject
+
+- (void)start;
```

Comment:
This is the saddest class ever. Should it be possible to stop the `APKServer`? If not, is it reasonable to create a server and not start it? If not, shouldn't it just start the moment you initialize it?

Reply:
Will make it start on creation